_posts/2023-02-28-admin-panels-for-index-operations.md

```diff
@@ -0,0 +1,144 @@
+
+The source index must block write operations, which means that you need to set the `index.blocks.write` setting in the source index to `true`. If the source index is not set to block write operations, you can choose the **Block write operations** button to set the `index.blocks.write` setting to `true` as seen in the following image.
+
+![Image: Shrink-block-write]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-block-write.jpg){:.img-fluid }
+
+The source index must also be open. If the source index is closed, the shrink operation will fail. You can choose the **Open** button to open the index as seen in the following image. This may take additional time to complete, and the index will be in the red health status while opening.
+
+![Image: Shrink-open]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-open.jpg){:.img-fluid }
+
+A copy of every shard in the source index must reside on the same node. You can move a copy of every shard to one node manually or use the shard allocation filter to do so. However, this is not required when the cluster has only one data node or if the replica count of the source index is equal to the number of data nodes minus one. In these cases, a copy of every shard in the source index resides on the same node. If a copy of every shard in the source index does not reside on the same node, you can update the `index.routing.allocation.require._name` setting of the source index to move shards to one node automatically.
+
+You can update this setting in the source index's detail page as seen in the following image.
+![Image: Shrink-move-shards]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-move-shards.jpg){:.img-fluid }
+
+Or you can submit the following request with `Dev Tools`:
+
+```
+PUT test-1/_settings
+{
+"index.routing.allocation.require._name":"node1"
+}
+```
+
+After ensuring that all prerequisites are met, you can fill out the input form on the **Shrink index** page. Specify a name, the primary shard count, and the replica count for the new shrunken index as seen in the following image. You can also specify new aliases or select existing aliases to attach to the new shrunken index.
+
+![Image: Shrink-configure]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-configure.jpg){:.img-fluid }
+
+All of the settings specified above will also be shown in the JSON editor under the **Advanced settings** section. You can also specify additional index settings for the new shrunken index in the JSON editor. For example, you can set both `index.routing.allocation.require._name` and `index.blocks.write` to `null` as seen in the following image. This will clear the allocation requirement and the index write block, which are copied from the source index.
```

Comment:
```suggestion
All of the specified settings will also be shown in the JSON editor under the **Advanced settings** section. You can also specify additional index settings for the new shrunken index in the JSON editor. For example, you can set both `index.routing.allocation.require._name` and `index.blocks.write` to `null`, as shown in the following image. This will clear the allocation requirement and the index write block, which are copied from the source index.
```